client/a8c-for-agencies/components/site-configurations-modal/index.tsx

```diff
@@ -109,6 +109,14 @@ export default function SiteConfigurationsModal( {
 						)
 					)
 				);
+			} else if ( error.code === 'email_unverified' ) {
+				dispatch(
+					errorNotice(
+						translate(
+							'Please verify your email address by clicking the link we sent to your inbox.'
+						)
+					)
+				);
```

Comment:
~Maybe add `id` and `duration` to the notice here?~

I think this ok, as we want them to verify the email! 

```suggestion
				dispatch(
					errorNotice(
						translate(
							'Please verify your email address by clicking the link we sent to your inbox.'
						),
						{
							id: 'site-configurations-modal-email-unverified',
							duration: 10000,
						}
					)
				);
```